contracts/NFTStorefrontV2.cdc

```diff
@@ -742,6 +759,30 @@ pub contract NFTStorefrontV2 {
             destroy listing
         }
 
+        /// cleanupGhostListings
+        /// Allow anyone to cleanup ghost listings
+        /// Listings will become ghost listings if stored provider capability doesn't hold
+        /// the NFT anymore.
+        ///
+        /// @param listingResourceID ID of the listing resource which would get removed if it become ghost listing.
+        pub fun cleanupGhostListings(listingResourceID: UInt64) {
```

Comment:
Why is this new code? Did it not exist already?

Reply:
This is from main, but I'm targetting the existing `stable-cadence` branch. So not technically new code, but new to the base branch